setup.py

```diff
@@ -36,7 +36,7 @@
             ]
         },
     install_requires=[
-        'OpenFisca-Core >= 4.1.5.dev1, < 5.0',
+        'OpenFisca-Core == 4.2.1.dev0',
```

Comment:
Does this mean that with `4.2.1` it doesn't work?

What is the version 4.2.1.dev0?